src/web_app/js/util.js

```diff
@@ -89,17 +85,12 @@ function requestTurnServers(turnRequestUrl, turnTransports) {
       }
       // Filter the TURN URLs to only use the desired transport, if specified.
       if (turnTransports.length > 0) {
-        filterTurnUrls(turnServerResponse.uris, turnTransports);
+        for (var i = 0; i < turnServerResponse.iceServers.length; i++) {
+          filterTurnUrls(turnServerResponse.iceServers[i].urls, turnTransports);
```

Comment:
Right, did not think it would here as we use our turn server but people can ofc specify a turn server in the URL parameters or by using their own turn server provider. Sorry for not bringing over the lessons learnt.